com.unity.render-pipelines.universal/Shaders/LitInput.hlsl

```diff
@@ -133,6 +149,20 @@ void ApplyPerPixelDisplacement(half3 viewDirTS, inout float2 uv)
     ApplyPerPixelDisplacement(TEXTURE2D_ARGS(_ParallaxMap, sampler_ParallaxMap), viewDirTS, _Parallax, uv);
 }
 
+// Used for scaling detail albedo. Main features:
+// - Depending if detailAlbedo brightens or darkens, scale magnifies effect.
+// - No effect is applied if detailAlbedo is 0.5.
+half3 ScaleDetailAlbedo(half3 detailAlbedo, half scale)
+{
+    // detailAlbedo = detailAlbedo * 2.0h - 1.0h;
+    // detailAlbedo *= _DetailAlbedoMapScale;
+    // detailAlbedo = detailAlbedo * 0.5h + 0.5h;
+    // return detailAlbedo * 2.0f;
+
+    // A bit more optimized
+    return 2.0h * detailAlbedo * scale - scale + 1.0h;
```

Comment:
I vaguely remember something about the `h` not working on some platform. you can achieve a generic way by doing `real(2.0)`

Reply:
I can change it, but I saw we have usage of h all over our shader codebase. Even searching *.0h hits 80 matches in urp package.